rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/nodesource/infrastructure/HostsFileBasedInfrastructureManager.java

```diff
@@ -558,7 +558,7 @@ protected String generateNodeName(HostTracker hostTracker) {
             String configuredAddress = hostTracker.getConfiguredAddress();
             String compliantAddress = convertToCompliantAddress(configuredAddress);
             return nodeSource.getName() + COMPLIANT_ADDRESS_DELIMITER + compliantAddress + COMPLIANT_ADDRESS_DELIMITER +
-                   ProActiveCounter.getUniqID();
+                   0;
```

Comment:
why `0` ?  and why not removing `COMPLIANT_ADDRESS_DELIMITER + 0`

Reply:
@mboussaa you need to look at HostsFileBased class to understand. COMPLIANT_ADDRESS_DELIMITER is used to extract the host name using a regexp. If I remove it , it will not be able to extract the host name. If I don't put 0, the node name will end with "__" which is ugly.